test/isIndexed.js

```diff
@@ -0,0 +1,28 @@
+import { assert } from 'chai';
+
+import * as RA from '../src';
+
+describe('isIndexed', function() {
+  context('given a `String` value', function() {
+    specify('should return true', function() {
+      assert.isTrue(RA.isIndexed('test'));
+      assert.isTrue(RA.isIndexed(Object('test')));
+    });
+  });
+
+  context('given array value', function() {
+    specify('should return true', function() {
+      assert.isTrue(RA.isIndexed([1, 2]));
```

Comment:
Let's add empty array check
```suggestion
      assert.isTrue(RA.isIndexed([]));
      assert.isTrue(RA.isIndexed([1, 2]));
```